src/components/Tooltip.scss

```diff
@@ -19,3 +19,8 @@
 		width: 0;
 	}
 }
+
+.pe-tooltip,
+.pe-tooltip * {
+	box-sizing: border-box;
```

Comment:
just sharing:
in mobile-wiki there is
https://github.com/Wikia/mobile-wiki/blob/dev/app/styles/base/_box-sizing.scss
this is a nice default that is easily overridden when needed